data/map altera.txt

```diff
@@ -4292,56 +4292,56 @@ system Zephyrius
-	description `	The buildings have few windows and large air-conditioners on their roofs, as well as a garage, to reduce the heat and the potential damage done by it. To take it a step further, whoever designed the initial building strategy, appears to have thought of everything, as even the tiniest building is connected to a network by either an overground glass tunnel or an underground path.`
-	spaceport `The spaceport is located in a hill, right next to the city, with what looks like a landing strip leading the spaceships into the hill. Even though the spaceport is located underground, it doesn't feel cramped at all and considering that this is a mining world's spaceport, it is suprisingly clean and most certainly less dusty than outside. It's location underground has resulted in a lower temperature and more efficient cooling, leading to an almost pleasant temperature. However, you don't have much of a view down here, not like it's better above the ground anyway.`
+	description `Dakara is a very rocky and sandy world, used almost exclusively for mining purposes. It used to be far greener, but ended up a bit too close to the sun, resulting in higher temperatures and little rainfall. Members of the resident population can be easily recognized by their darker skintone, compared to the usual, almost white, Alteran skintone.`
+	description `	The buildings have only few windows and large air-conditioners and garages on their roofs, to reduce heat and radiation damage coming from the sun. To take it a step further, the designeres of the initial building strategy ensured that even the tiniest building can be reached either by a protected above-ground walkway or by an underground path.`
+	spaceport `The spaceport is located in a hill, right next to the city, with what looks like a landing strip leading the spaceships into the hill. Even though the spaceport is located underground, it doesn't feel cramped at all and considering that this is a mining world's spaceport, it is suprisingly clean and most certainly less dusty than outside. Its underground location supports a more efficient cooling, resulting in an almost pleasant temperature. Unsuprisingly, the view is rather limited.`
 
 planet Draconis
 	landscape land/city14-sfiera
-	description `Draconis is the eastern co-capital of the Alteran Empire, it was initially settled as a mining world, as it was a mineral rich world, later on, Draconium was discovered on this world, making it one of the few worlds with known deposits of Draconium. There are multiple large cities located on the main continent and some of the large mining complexes can even be seen from space.`
-	description `	At this point, it is no longer just a mining world, but has been taken as the seat of one of the progenitors, who rules the eastern space from here. It can be counted as one of the most populated and economically strongest worlds of the Alterans.`
```

Comment:
These are mostly grammatical corrections. If you don't like "appointed", "chosen" would be a good alternative.